src/main/java/net/sf/rails/game/Bonus.java

```diff
@@ -2,13 +2,13 @@
 
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import net.sf.rails.algorithms.NetworkVertex;
 import net.sf.rails.algorithms.RevenueAdapter;
 import net.sf.rails.algorithms.RevenueBonus;
 import net.sf.rails.algorithms.RevenueStaticModifier;
-
-
+import net.sf.rails.game.state.Observable;
```

Comment:
Why are these imports added when nothing else changed?

Reply:
Because by accident I also pulled this file out of the set of changes while pulling the other examples. Nothing to see here ;)